src/djinterop/impl/database_impl.hpp

```diff
@@ -20,6 +20,7 @@
 #include <optional>
 #include <string>
 #include <vector>
+#include <stdint.h>
```

Comment:
Hi Bart, thanks for the PR :smile:!

```suggestion
#include <cstdint>
#include <optional>
#include <string>
#include <vector>
```

`<stdint.h>` is the C library header.  Does the C++ header `<cstdint>` work for you?

(Also attempting to keep includes in vaguely alphabetical order - one day I'll add clang-format pre-commit hooks for this...)

Reply:
Yeah that worked as well, updated :+1: